docs/source/hardware/scavenger.rst

```diff
@@ -0,0 +1,69 @@
+
+.. _partition_names:
+
+**********
+Scavenger
+**********
+:sub:`Updated July, 2022`
+
+Introduction
+===================
+‘Scavenger’ is a new partition on discovery that allows the research community to use private resources 
+on the cluster when they are idle. ‘Scavenger’ has hardware not otherwise available to the general research 
+community and, in time, will double the resources to which users have access.
+
+When to use scavenger
```

Comment:
Capitalize Scavenger here, although it will be specified as `scavenger` (lower-case) in configuration and commands I gather.